frontend/components/panels/DesktopSectionPanel.js

```diff
@@ -116,6 +135,22 @@ export default class DesktopSectionPanel extends React.Component {
 
     const honorsCheck = this.props.section.honors ? <Icon name='check' /> : <Icon name='x' />;
 
+    let notifBox;
+
+    if (this.props.section.seatsRemaining === 0) {
+      notifBox = (
+        <div data-tip='Sign up for notifications for this section' className='inlineBlock'>
```

Comment:
Can you move this <div data-tip='... > into the < NotifCheckBox > file? Would make more sense to have this logic inside this file, especially if you passing this.props.section.seatsRemaining into < NotifCheckBox >